Assets/MRTK/SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -434,6 +434,13 @@ bool IMixedRealityNearPointer.TryGetNearGraspPoint(out Vector3 position)
             return false;
         }
 
+        /// <inheritdoc />
+        bool IMixedRealityNearPointer.TryGetNearGraspAxis(out Vector3 axis)
+        {
+            axis = transform.forward;
+            return false;
```

Comment:
Since this is returning data that seems to match the direction this pointer is pointing, should it:
```suggestion
            return true;
```